common/djangoapps/third_party_auth/pipeline.py

```diff
@@ -192,6 +177,12 @@ class AuthEntryError(AuthException):
     """
 
 
+class NotActivatedException(AuthException):
+    """ Raised when a user tries to login to an unverified account """
+    def __str__(self):
+        return _('This account has not yet been activated. An activation email has been re-sent.')
```

Comment:
Is it possible to include the e-mail address where we're sending the message here? If so, I'd say "This account has not yet been activated. An activation email has been re-sent to `<address>`." If not, I'd just say "This account has not yet been activated. An activation email has been re-sent to the address associated with this account."